src/main/riddl/Organization/organizationMessages.riddl

```diff
@@ -145,3 +145,39 @@ result OrganizationInfo is {
   orgId: OrganizationId,
   info: OrganizationContext.Info
 }
+query GetOrganizationParent is {orgId: OrganizationId}
+result OrganizationParent is {
+  orgId: OrganizationId,
+  parent: OrganizationId?
+}
+query GetOrganizationMembers is {orgId: OrganizationId}
+result OrganizationMembers is {
+  orgId: OrganizationId,
+  members: MemberId+
+}
+query GetOrganizationOwners is {orgId: OrganizationId}
+result OrganizationOwners is {
+  orgId: OrganizationId,
+  owners: MemberId+
+}
+query GetOrganizationContacts is {orgId: OrganizationId}
+result OrganizationContacts is {
+  orgId: OrganizationId,
+  contacts: Contact*
+}
+query GetOrganizationMeta is {orgId: OrganizationId}
+result OrganizationMeta is {
+  orgId: OrganizationId,
+  meta: OrganizationContext.MetaInfo?
+}
+query GetOrganizationState is {orgId: OrganizationId}
```

Comment:
This was confusing to me. When I first read this I was thinking Open, Closed, etc. But I can see by the result that you are thinking all the attributes that define that organization. Wouldn't those attributes potentially change by state (as in Open, closed)?

Reply:
I get your point. But just to be pedantic: + would be required (one or more)
I think optionality would be * (zero or more) or ? (zero or one).

I am not sure if this distinction would be confusing to others or not. The way I was thinking in the first iteration:
getState = returns active, inactive, suspended, terminated, etc.
getInfo = returns data about the organization (info, metaInfo, etc.)

Reply:
Have changed the result to only return `currentState` contained in `metaInfo`